pkg/virt-operator/kubevirt_test.go

```diff
@@ -2746,9 +2746,9 @@ var _ = Describe("KubeVirt Operator", func() {
 
 			kvTestData.controller.Execute()
 
-			Expect(len(kvTestData.controller.stores.RoleCache.List())).To(Equal(2))
-			Expect(len(kvTestData.controller.stores.RoleBindingCache.List())).To(Equal(2))
-			Expect(len(kvTestData.controller.stores.ServiceMonitorCache.List())).To(Equal(0))
+			Expect(kvTestData.controller.stores.RoleCache.List()).To(HaveLen(2))
+			Expect(kvTestData.controller.stores.RoleBindingCache.List()).To(HaveLen(2))
+			Expect(kvTestData.controller.stores.ServiceMonitorCache.List()).To(HaveLen(0))
```

Comment:
You have already used `BeEmpty` as part of this commit. Is this one missed?

Reply:
Thanks. Fixed.